ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@
       setStreaming(false)
     }
```

Comment:
Similarly, this line uses optional chaining to access `getBoundingClientRect()` on `interpretEndRef.current`. The variable `dotToTopHeight` will hold the top position of the element referenced by `interpretEndRef`.